### ISSUE TITLE
[COGNITION] Change style of review card field cancel button

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#8096]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Review cards display formatted information when the data is prefilled, along with an "Edit" button, otherwise the associated form elements are shown allowing the user to edit & update the content.

![Screen Shot 2020-04-16 at 2 44 21 PM](https://user-images.githubusercontent.com/136959/79499650-ea6b4a80-7ff0-11ea-8a32-37c444137c78.png)

In edit mode, a "Cancel" button was recently added, but was styled as a secondary button.

![Screen Shot 2020-04-16 at 2 31 01 PM](https://user-images.githubusercontent.com/136959/79499496-abd59000-7ff0-11ea-9403-abf9147ece2e.png)

Now, due to cognitive considerations, the recommendation is to change the style of the "Cancel" button to appear as a link. Additionally the "Done" text should be changed to "Save".

## Acceptance Criteria

The "Save" (previously "Done") button should not be confused with the "Cancel" action, ever.

## Steps to Recreate

1. Go to https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/
1. Log in as user 228
1. Continue until you get to the second part of step 1 - the url ending in `/contact-information`
1. Press the "Edit" button
1. You should now see the "Save" button & "Cancel" link.

## Definition of Done

- [ ] Review card, in edit mode, displays a "Save" button
- [ ] Review card, in edit mode, displays a "Cancel" link
